tests/openfl/component/assigner/test_assigner.py

```diff
@@ -46,13 +44,23 @@ def test_get_aggregation_type_for_task(assigner):
 def test_get_all_tasks_for_round(assigner):
     """Assert that assigner tasks object is list."""
     assigner = Assigner(None, None, None)
-    tasks = assigner.get_all_tasks_for_round('test')
+    tasks = assigner.get_all_tasks_for_round("test")
 
     assert isinstance(tasks, list)
 
+def test_default_task_group(assigner):
+    """Assert that by default learning task_group is assigned."""
+    assigner = Assigner(None,None,None)
+    assert assigner.selected_task_group == 'learning'
```

Comment:
For completeness, could you also add a unit test where `assigner.selected_task_group == 'evaluation'`?